### PR TITLE
[AKS] Adding serviceType to values for rendering Notes

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: dotnet/aspnetcore/kubernetes/Application/aspnet-core-dotnet-core/charts/sampleapp/values.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ imagePullSecrets: []
   # This uses credentials from secret "myRegistryKeySecretName".
   # - name: myRegistryKeySecretName
 service:
+  type: LoadBalancer
```
Comment (sachinma) [inline dotnet/aspnetcore/kubernetes/Application/aspnet-core-dotnet-core/charts/sampleapp/values.yaml]: why is this required? I thought this is a derived value

File: java/jsf/kubernetes/Application/charts/sampleapp/values.yaml [hunk 2]
```diff
@@ -2,10 +2,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 fullnameOverride: sampleapp
-replicaCount: 1
+replicaCount: 2
```
Comment (sachinma) [inline java/jsf/kubernetes/Application/charts/sampleapp/values.yaml]: why do we need to increase the replica count.

Comment (ds-ms) [reply java/jsf/kubernetes/Application/charts/sampleapp/values.yaml]: Just making the count same for every sample app
